File: MeetingWithJava/JAVA/STRING/FindNum.java
public class FindNum {

  public static void main(String[] args) {
    String s1 = "Aas0hi23sh89";
    String s2 = "";

    for (int i = 0; i < s1.length(); i++) {
      if (s1.charAt(i) >= 48 && s1.charAt(i) <= 57) {
        s2 += s1.charAt(i);
      }
    }
    System.out.println(s2);
  }
}

File: MeetingWithJS/FINAL_JS_REVESION/JS/String22222/FindNum.js
(function (){
    let str = 'Aashish1234567 A890jmera';
    let findNum = '', count = 0;

    for(let i = 0; i < str.length; i++){
        if(str.charCodeAt(i) >= 48 && str.charCodeAt(i) <= 57){
            findNum += str.charAt(i);
            count++;
        }
    }

    console.log('Total digit is: ', count ,' and digit is: ', findNum);
})();